cookieapp.js

```diff
@@ -0,0 +1,200 @@
+'use strict'
+
+//First store object
+var pikeStore = {
+
+  storeName: '1st and Pike',
+  storeHours: ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'],
+  salesHr: [],
+  minCustomerHr: 23,
+  maxCustomerHr: 65,
+  avgSalesHr: 6.3,
+
+// Calculates random number of customers, which a loop is then used
+//loop runs to calculate number of sales based on random customer output
+  randomCustomerHr: function() {
+    return Math.ceil(Math.random() * (this.maxCustomerHr / this.minCustomerHr));
+  },
+
+  hourlySales: function() {
+    for (var i = 0; i < this.storeHours.length; i++){
+      this.salesHr.push(Math.ceil(Math.random()*(this.avgSalesHr * this.randomCustomerHr())))[i];
+      // console.log(this.salesHr);
+    }
+  },
+}
+
+//keyvalue pairs stored in objects & sales data calculated above
+//are assigned to variables
+
+var getStoreHours = pikeStore.storeHours;
+var generateSalesData = pikeStore.hourlySales();
+var getHourlycookieSales = pikeStore.salesHr;
```

Comment:
'get' is usually used in a function name as it donates an action is occurring (getNames, getHours, etc) while a variable should be a noun (storeHours, custName, etc).

You are not modifying the value stored i pikeStore.salesHr at all, instead of binding to a variable you can simply refer to the object property below